Formula/i386-elf-gdb.rb

```diff
@@ -0,0 +1,20 @@
+class I386ElfGdb < Formula
+  desc "The GNU Project debugger for i386-elf cross development"
+  homepage "https://www.gnu.org/software/gdb/"
+  url "https://ftp.gnu.org/gnu/gdb/gdb-8.2.tar.xz"
+  sha256 "c3a441a29c7c89720b734e5a9c6289c0a06be7e0c76ef538f7bbcef389347c39"
+
+  def install
+    mkdir "i386-elf-gdb-build" do
+      system "../configure", "--target=i386-elf",
+                             "--prefix=#{prefix}",
+                             "--disable-werror"
+      system "make"
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "#{bin}/i386-elf-gdb", "#{bin}/i386-elf-gdb", "-configuration"
```

Comment:
What is that test doing?

Reply:
I have referenced [Homebrew-gdb](https://github.com/Homebrew/homebrew-core/blob/734b459f7bc414e58f55f6d6bc386ede79f304e9/Formula/gdb.rb) line 97

Reply:
I'd still like to know what this is doing. Can we match anything from that output?

Reply:
It will print details about GDB configuration, include compile options and path. And I think it would be enough. I also tried to match something got with the command `list`, but gdb had show most useless messages as the following picture
<img width="829" alt="2018-11-28 12 40 32" src="https://user-images.githubusercontent.com/12967882/49096910-97331380-f2a6-11e8-8c57-0b72e8cc4d77.png">
If we have to match something, does those useless messages have an impact?